src/components/web3/WalletSelectModal.vue

```diff
@@ -4,16 +4,40 @@
     @close="$emit('close')"
     title="Connect to a wallet"
   >
+    <p class="pb-3 text-sm">
+      By connecting a wallet, I agree to Balancer Labs’
```

Comment:
Let's move the text in this component to the default.json locales file.

Reply:
Added these, plus the 'New to Ethereum' section below to the default.json locales file